rclcpp/test/test_function_traits.cpp

```diff
@@ -142,15 +143,15 @@ TEST(TestFunctionTraits, arity) {
       return 0;
     };
 
-  auto lambda_one_int = [](int) {
+  auto lambda_one_int = [](int /*one*/) {
```

Comment:
I didn't know that you changed it because of cpplint. You can keep it as-is then.